### DISCUSSION TITLE
Is it possible to set a button toggle?

### DISCUSSION THREAD

**Discussion by kalamalka** [Snoothy/UCR#252] [Q&A]
Greetings. I am looking for a way to create a toggle for a button that requires constant input. For example, in the game I am playing the cross button is used as a modifier to make the player run instead of walk. I would like it so that I press this button once and the button state is registered as pressed down until I press the button again.

Does UCR have this functionality?

**Comment by technikker** [comment]
I am also having this problem.  The intuitive thing would be a "filter to button" option, but this does not appear to be possible.  I've tried hacking around, including having an inverted "button to button" assignment on a key that is never pressed to create an "always true" case, but this just produces strange and unexpected behavior (you have to press the button that you never press each time the filter is enabled to get it to update)
